frontend/src/store/sagas/students.js

```diff
@@ -0,0 +1,52 @@
+import {
+  put,
+} from 'redux-saga/effects'
+import { fetchStudent, fetchStudents, createStudent, editStudent } from '../api/students'
+import {
+
+  GET_STUDENT_INFO_SUCCESS,
+  GET_STUDENT_INFO_FAILURE,
+  GET_ALL_STUDENTS_SUCCESS,
+  GET_ALL_STUDENTS_FAILURE,
+  SUBMIT_NEW_STUDENT_FORM_FAILURE,
+  SUBMIT_NEW_STUDENT_FORM_SUCCESS,
+  SUBMIT_EDIT_STUDENT_FORM_SUCCESS,
+  SUBMIT_EDIT_STUDENT_FORM_FAILURE,
+} from '../actions/students'
+
+export function* fetchStudentInfo(action) {
+  try {
+    const json = yield fetchStudent(action.studentId)
+    yield put({ type: GET_STUDENT_INFO_SUCCESS, studentdonator: json.data.studentdonator })
```

Comment:
Since we now support flash messages, can we please also add flash messages here too? 🙇‍♂️